lib/connection.js

```diff
@@ -58,6 +58,10 @@ function Connection(transport, server, client) {
 
   this._heartbeatCallbackInstance = null;
 
+  this._emitError = (error) => {
```

Comment:
Can you please add a comment explaining the reason of this being created in the constructor instead of on the prototype? It is not immediately obvious until one reads the `callMethod()` code.